include/cudaq/Optimizer/Dialect/Quake/QuakeOps.td

```diff
@@ -852,15 +852,25 @@ class QuakeOperator<string mnemonic, list<Trait> traits = [],
                    "mlir::ValueRange":$controls,
                    "mlir::ValueRange":$targets,
                    "mlir::DenseBoolArrayAttr":$negates), [{
-      return build($_builder, $_state, mlir::TypeRange{}, is_adj, parameters,
+      return build($_builder, $_state, static_cast<bool>(is_adj), parameters,
```

Comment:
Probably want to just add a new `bool` builder overload that's missing the return type.

Reply:
> Probably want to just add a new `bool` builder overload that's missing the return type.

Is that what is in the next few lines?